civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -1601,6 +1603,12 @@ function civihr_employee_portal_menu() {
     'type' => MENU_CALLBACK,
   ];
 
+  $items['emergency_contacts/%/delete'] = [
+    'page callback' => 'civihr_employee_portal_emergency_contact_delete',
+    'page arguments' => [1],
+    'access callback' => TRUE,
```

Comment:
What does this mean in terms of permissions? Who will have access to this page?

Reply:
Everyone, I thought about adding a permission callback, but it would just be duplicating the permission check in the service. I can do that if you want, but it would mean we check "is this the emergency contact owner" both before accessing the page and during page execution